### PR TITLE
feature/DTFS2-4187: adds ability to Abandon a deal

### PR DESCRIPTION
**Pull request by pbradley-f4**

### CODE REVIEW

File: gef-ui/templates/application-abandon.njk [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        You’ll still be able to find this {% if application.isAutomaticCover %} notice {% else %} application {% endif %} in the abandoned folder. 
+      </p>
+    </div>
+
+    <div class="govuk-grid-column-one-half-from-desktop">
+      <form
+        method="post"
+        data-cy="form">
+
+        <p class="govuk-button-group govuk-!-margin-top-7" style="vertical-align: middle">
```
Comment (ttbarnes) [inline gef-ui/templates/application-abandon.njk:40]: is the inline style needed?

File: gef-ui/server/controllers/application-abandon/index.test.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  ],
+});
+
+const MockFacilityResponse = () => {
+  const res = {};
+  res.status = 'IN_PROGRESS';
+  res.data = [];
+  return res;
+};
+
```
Comment (ttbarnes) [inline gef-ui/server/controllers/application-abandon/index.test.js:75]: Not for this PR, seems to be a lot of duplication of the mock responses, would be nice to move into one shared file

**Review by ttbarnes — DISMISSED** [review]
non blocking comments otherwise LGTM

File: gef-ui/server/controllers/application-abandon/index.test.js (hunk 2 above)
Comment (pbradley-f4) [reply gef-ui/server/controllers/application-abandon/index.test.js:75]: yeah I will refactor that later when we have time

File: gef-ui/templates/application-abandon.njk (hunk 1 above)
Comment (pbradley-f4) [reply gef-ui/templates/application-abandon.njk:40]: good catch ..  I think that was leftover from when it was a link. button-group resolves this so will remove

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**